### DISCUSSION TITLE
API routes example(s) with `App Router` like <https://github.com/vercel/next.js/blob/canary/examples/api-routes>

### DISCUSSION THREAD

**Discussion by MikeChongCan** [vercel/next.js#49268] [Help] [answered]
### Summary

<https://github.com/vercel/next.js/blob/canary/examples/api-routes/pages/api/people/index.ts> is a great example of API routes in the old pages mode.

However, I haven't found an equivalent of it while using the `App Router` mode.

**Comment by icyJoseph** [comment, marked as answer]
For the time being this should be good enough:

- Playground app which has most of the Next.js features https://github.com/vercel/app-playground/tree/main/app/api
- Extensive docs https://nextjs.org/docs/app/building-your-application/routing/router-handlers

**Reply by MikeChongCan** [reply]
Thank you!
